Gemfile

```diff
@@ -36,3 +34,11 @@ group :development do
     gem "rubocop-shopify", "2.10.1"
   end
 end
+
+# If Psych doesn't build, you can disable this group locally by running
+# `bundle config set --local without rdoc`
+# Then re-run `bundle install`.
+group :rdoc do
+  gem "rdoc", "6.5.0"
+  gem "psych", "~> 5.0" # psych 5 isn't building in places yet https://github.com/ruby/setup-ruby/issues/409
```

Comment:
I think we should remove the `psych` line now. It was originally added in 6a3254cd to pin to `~> 4.0` but that was later updated by dependabot to `~> 5.0` in c7a78029. We actually don't care about Psych as long as it stays out of our way!